app/controllers/stories_controller.rb

```diff
@@ -126,7 +126,26 @@ def handle_base_index
     @stories = ArticleDecorator.decorate_collection(@stories)
     set_surrogate_key_header "main_app_home_page"
     response.headers["Surrogate-Control"] = "max-age=600, stale-while-revalidate=30, stale-if-error=86400"
-    render template: "articles/index"
+
+    respond_to do |format|
+      format.html do
+        render template: "articles/index"
+      end
+      format.json do
+        render json: @stories.to_json(
+          only: %i[
```

Comment:
Can we please pull the two arrays out into constants?

Reply:
I was limited by where I could put the constant (`Articles.rb` is too long), so I popped it into the Jbuilder template.